integration-cli/docker_cli_import_test.go

```diff
@@ -41,3 +42,49 @@ func TestImportDisplay(t *testing.T) {
 
 	logDone("import - display is fine, imported image runs")
 }
+
+func TestImportFile(t *testing.T) {
+	runCmd := exec.Command(dockerBinary, "run", "-d", "busybox", "true")
+	out, _, err := runCommandWithOutput(runCmd)
+	if err != nil {
+		t.Fatal("failed to create a container", out, err)
+	}
+	cleanedContainerID := stripTrailingCharacters(out)
+	defer deleteContainer(cleanedContainerID)
+
+	runCmd = exec.Command(dockerBinary, "export", cleanedContainerID)
+	out, _, err = runCommandWithOutput(runCmd)
```

Comment:
This scares me a little to potentially put a ton of data into a string.  It might be better to just redirect stdout of the cmd directly to the file as part of executing t.